http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/ApacheSdkHttpService.java

```diff
@@ -25,6 +26,6 @@
 
     @Override
     public SdkHttpClientFactory createHttpClientFactory() {
-        return ApacheSdkHttpClientFactory.builder().build();
+        return ApacheSdkHttpClientFactory.builder().apacheHttpClientFactory(new ApacheHttpClientFactory()).build();
```

Comment:
If we are going to keep the configuration this way, I'd suggest a name change. Bit odd that ApacheSdkHttpClientFactory has a method called apacheHttpClientFactory. At first look you would think the class is taking an instance of itself in.